hardware/1Wire/1WireByKernel.cpp

```diff
@@ -233,8 +233,11 @@ void C1WireByKernel::GetDevices(/*out*/std::vector<_t1WireDevice>& devices) cons
 {
    // If devices not initialized (ie : at least one time read), return an empty list
    // to not update Domoticz with inconsistent values
-   if (!m_AllDevicesInitialized)
-      return;
+
+	while (!m_AllDevicesInitialized)
```

Comment:
will not deadlock
GetDevices() called from 1wire.cpp will wait endlessly .... but without deadlock
But m_AllDevicesInitialized set to true always after max few seconds (in ThreadFunction)